docs/resources/config_context.md

```diff
@@ -177,9 +177,11 @@ resource "rafay_config_context" "config-context-example" {
 
 ***Optional***
 
-- `restricted_values` (List of String) If the override type is restricted, specify the values it is restricted to
-- `type` (String) Specify the type of override this variable supports, Available types are `allowed`, `notallowed`, `restricted`
+- `display_overridden` (Boolean) Display overridden values on UI
+- `restricted_key_values` (Map of String) If the override type is restricted_key_values, values it is restricted to
+- `restricted_values` (List of String) If the override type is restricted, values it is restricted to
 - `selectors` (List of String) Used to alias a variable and restrict the override scope
+- `type` (String) Specify the type of ovverride this variable supports
```

Comment:
Need to specify types we support, like Available types are `allowed`, `notallowed`, `restricted`